ctru-rs/src/services/ac.rs

```diff
@@ -74,24 +74,32 @@ impl Ac {
     ///
     /// let ac = Ac::new()?;
     ///
-    /// println!("Wi-Fi connected: {}", ac.get_wifi_status()?);
+    /// println!("Wi-Fi status: {:?}", ac.get_wifi_status()?);
     /// #
     /// # Ok(())
     /// # }
     /// ```
     #[doc(alias = "ACU_GetWifiStatus")]
-    pub fn get_wifi_status(&self) -> crate::Result<bool> {
+    pub fn get_wifi_status(&self) -> crate::Result<NetworkStatus> {
         unsafe {
             let mut ret = 0u32;
```

Comment:
Make sure you `cargo fmt`, looks like this failed the lint check https://github.com/rust3ds/ctru-rs/actions/runs/8026159593/job/21928158019?pr=140#step:6:83